easybuild/tools/config.py

```diff
@@ -109,6 +109,7 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
         'only_blocks',
         'optarch',
         'parallel',
+        'rebuild',
```

Comment:
this should be added to the `False` list rather than the `None` list

in practice, it's not going to make a difference probably, but it's better to do it 'right'

@cfenoy: please fix this, by moving `rebuild` into the `False` list below?